### DISCUSSION TITLE
Can you make 500 error page for api without custom server?

### DISCUSSION THREAD

**Discussion by VityaSchel** [vercel/next.js#26009] [Help]
Is there a way to create 500 error page with next.config.js? I want my api to always return application/json, not plain 'Internal server error'

**Comment by huksley** [comment]
I have a same question. Is it possible?
Officially provided example https://nextjs.org/docs/going-to-production#error-handling does not handle this.

**Reply by VityaSchel** [reply]
I did it by using middlewares, just put try/catch in one of them and then respond with json
